05-visualisation-ggplot2.Rmd

```diff
@@ -185,6 +185,7 @@ sections and points are counted wihin hexagons. The number of points per hexagon
 is encoded by color.
 
 ```{r}
+library(hexbin)
```

Comment:
hexbin is required for that whole plot. It's not just a color scheme. Since it does require an additional library, I just took that whole example out.